js/src/forum/utils/extendTagOverview.js

```diff
@@ -37,3 +39,39 @@ export default function extendTagOverview() {
     return items;
   });
 }
+
+function findChild(parent, childClass, recursive = false) {
```

Comment:
I think there should be a `maxDepth` option here so we don't recurse through too many layers of children.

It might not be an issue right now, but I'd feel more comfortable with such an option.

Reply:
done @davwheat